feature/bgp/linkbandwidth_community/otg_tests/linkbandwidth_aggregation/README.md

```diff
@@ -0,0 +1,84 @@
+# RT-7.6: BGP Link Bandwidth Community - Aggregation
+
+## Summary
+
+This test verifies Link-bandwidth (LBW) extended community aggregation feature by DUT.
+
+## Testbed type
+
+*  [`featureprofiles/topologies/atedut_2.testbed`](https://github.com/openconfig/featureprofiles/blob/main/topologies/atedut_2.testbed)
+
+## Procedure
+
+### Test environment setup
+
+    ```
+                                          |         |
+    [ 64x eBGP ] --- [ ATE Port 1 ] ----  |   DUT   | ---- [ ATE Port 2 ]
+                                          |         |
+    ```
+
+### RT-7.6.1: Verify LBW aggregation to eBGP peer
+
+* Configure DUT with routed ports on DUT.
+* Configure 64x eBGP peers on ATE Port 1 interface in peering group UPSTREAM.
+* Configure a single eBGP peer on ATE Port 2 interface in peering group DOWNSTREAM.
+* Enable BGP LBW receive for peering group UPSTREAM.
+* Enable BGP LBW send for peering group DOWNSTREAM. Enable Link Bandwidth aggregation feature.
+
+**TODO:** [Cumulative Link Bandwidth feature](https://datatracker.ietf.org/doc/draft-ietf-bess-ebgp-dmz/) is not currently modeled in OC. Cisco config example: `ebgp-send-extcommunity-dmz cumulative`
+
+
+2. Advertise the same test prefix from ATE from all UPSTREAM peers with LBW community:
+  * 32 peers - 10Mbps
+  * 16 peers - 20Mbps
+  * 8 peers - 40Mbps
+  * 8 peers - 80Mbps
+
+3. Verify that DUT advertises the test route to DOWNSTREAM eBGP peer with aggregated bandwidth community of 1600Mbps.
+
+### RT-7.6.2: Verify LBW changes.
+Using RT-7.6.1 set up conduct following changes:
+
+1) Disable 32 peers advertising 10Mpbs bandwidth community.
+2) Verify that DUT advertises the test route to Upstream peer with aggregated bandwidth community of 1280Mbps.
+3) Re-enable 32 peers advertising 10Mpbs bandwidth community.
+4) Verify that DUT advertises the test route to Upstream peer with aggregated bandwidth community of 1600Mbps.
+
+### RT-7.6.3: Verify LBW aggregation to iBGP peer
+
+* Configure 64x eBGP peers on ATE Port 1 interface in peering group UPSTREAM.
+* Configure a single iBGP peer on ATE Port 2 interface in peering group DOWNSTREAM.
+* Enable BGP LBW receive for peering group UPSTREAM.
+* Enable BGP LBW send for peering group DOWNSTREAM. Enable Link Bandwidth aggregation feature.
+
+**TODO:** Link Bandwidth aggregation feature is not currently modeled in OC.
+
+2. Advertise the same test prefix from ATE from all UPSTREAM peer with LBW community:
+  * 32 peers - 10Mbps
+  * 16 peers - 20Mbps
+  * 8 peers - 40Mbps
+  * 8 peers - 80Mbps
+
+3. Verify that DUT advertises the test route to DOWNSTREAM iBGP peer with aggregated bandwidth community of 1600Mbps.
```

Comment:
It's probably ok to specify changing the testbed configuration BGP peer types from eBGP to iBGP and repeat test 7.6.1.  (That's probably how someone would write the code)